include/dba.php

```diff
@@ -504,6 +504,14 @@ function dbesc($str) {
 	}
 }
 
+function any_value_fallback($sql) {
```

Comment:
Done in https://github.com/friendica/friendica/pull/3323/commits/06f374b26b01674707c135843d0df5189a9e57cf